jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -21,9 +21,11 @@ def buildExclusions = [
   // Linux -------------------------------------------------
   [ /^centos6/,                       releaseType, lt(8)   ],
   [ /^centos6/,                       anyType,     gte(12) ],
-  [ /centos[67]-(arm)?(64|32)-gcc48/, anyType,     gte(10) ],
-  [ /centos[67]-(arm)?(64|32)-gcc6/,  anyType,     lt(10)  ],
-  [ /centos6-32-gcc6/,                releaseType, gte(10) ], // 32-bit linux for <10 only
+  [ /^centos[67]-(arm)?(64|32)-gcc48/,anyType,     gte(10) ],
+  [ /^centos[67]-(arm)?(64|32)-gcc6/, anyType,     lt(10)  ],
+  [ /^centos[67]-(arm)?(64|32)-gcc6/, anyType,     gte(14) ], // 14.x: gcc6 builds stop
+  [ /^centos7-(arm)?(64)-gcc8/,       anyType,     lt(14)  ], // 14.x: gcc8 builds start
```

Comment:
Here we are stopping using one set of labels after 14.x, and switching to a new set. I think its intentional to do  anyType, its not intended that we test with gcc6/aka devtoolset-6, is it?

Reply:
ack
Yes let's just leave it as a clean break even for testing. We can expand later if we see a need but we had had a problem with exploding permutation complexity _just because we can_ in the past, and let's try to resist that urge.

Reply:
I assume we still have some linux platforms building/testing on gcc6  thought right ?

Reply:
For 14.x, specifically? I don't know. For earlier releases, yes.

If you want I can take can try to find the compiler levels for all machines 14.x are tested on, but I spent 10 minutes looking, and its not documented anywhere AFAICT. Figuring it out would take a combination of finding all linux builds in https://ci.nodejs.org/view/Node.js%20Daily/job/node-daily-master/1899/, reading the console logs, then sshing into the machines (if they don't explicitly set the version, many don't, I have to look... for example ubuntu 18.04 is using gcc7, I just checked). If https://github.com/nodejs/node/pull/32715#issuecomment-611595712 lands, then the ssh won't be needed, just reading the top of all the console logs.

AIX builds on gcc6.

Reply:
From the CI for https://github.com/nodejs/node/pull/32715 (https://ci.nodejs.org/job/node-test-pull-request/30565/) the following Linux jobs are using gcc6:

- [debian9-64](https://ci.nodejs.org/job/node-test-commit-linux/34121/nodes=debian9-64/consoleText) is using 6.3.0
- [ubuntu1604-64](https://ci.nodejs.org/job/node-test-commit-linux/34121/nodes=ubuntu1604-64/consoleText) is using 6.5.0